test/ably/restrequest_test.py

```diff
@@ -81,7 +81,7 @@ def test_not_found(self):
     @dont_vary_protocol
     def test_error(self):
         params = {'limit': 'abc'}
-        result = self.ably.request('GET', '/channels/test/messages', params=params)
+        result = self.ably.request('GET', '/channels/rest-request/messages', params=params)
```

Comment:
This is likely not going to fix the "tests are run in parallel" problem, because it's still a static string. I suggest you create a random name for the channel. It's something we already do in [JS tests](https://github.com/ably/ably-js/blob/0b60fb298e72763544ea7f91737234e1822700f0/spec/browser/simple.test.js#L136).

Reply:
done